h/models/group.py

```diff
@@ -85,11 +84,15 @@ class Group(Base, mixins.Timestamps):
         'User', secondary='user_group', backref=sa.orm.backref(
             'groups', order_by='Group.name'))
 
-    def __init__(self, name, authority, creator, description=None):
+    def __init__(self, name, authority, creator, description=None,
+                 joinable_by=None, readable_by=None, writeable_by=None):
         self.name = name
         self.authority = authority
         self.description = description
         self.creator = creator
+        self.joinable_by = joinable_by
+        self.readable_by = readable_by
+        self.writeable_by = writeable_by
```

Comment:
I know, this can be fixed later :) I guess the reason is that the creator will automatically added to the members, plus not using `super` :)